pallets/rmrk-market/src/benchmarking.rs

```diff
@@ -184,8 +184,11 @@ benchmarks! {
 
 	accept_offer {
 		let caller: T::AccountId = whitelisted_caller();
-
 		let collection_index = 1;
+		// The balance is not set to `max_value` because the caller needs to
```

Comment:
We could add this to the `create_test_collection` instead of `BalanceOf::<T>::max_value()` this reduces repeating the command